packages/flutter/lib/src/widgets/unique_widget.dart

```diff
@@ -23,8 +23,8 @@ abstract class UniqueWidget<T extends State<StatefulWidget>> extends StatefulWid
   /// The [key] argument is required because it identifies the unique inflated
   /// instance of this widget.
   const UniqueWidget({
```

Comment:
```suggestion
  const UniqueWidget({required GlobalKey<T> super.key});
```

Might as well save a couple of lines 🙂